.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://registry.yarnpkg.com"
```

Comment:
Why this ?

Reply:
It's for our internal usage so we don't pollute `yarn.lock` with links to our private repo